apis/ec2/src/main/java/org/jclouds/ec2/compute/domain/EC2HardwareBuilder.java

```diff
@@ -429,6 +435,69 @@ public static EC2HardwareBuilder t2_large() {
             .processors(ImmutableList.of(new Processor(1.0, 0.4))).rootDeviceType(RootDeviceType.EBS);
    }
 
+   /**
+    * @see InstanceType#T3_NANO
+    */
+   public static EC2HardwareBuilder t3_nano() {
+      return new EC2HardwareBuilder(InstanceType.T3_NANO).t1()
+              .ram(512)
+              .processors(ImmutableList.of(new Processor(2.0, 1.0)));
```

Comment:
Where can I found the correct speed?

Reply:
AWS Instance Type reference page:
https://aws.amazon.com/ec2/instance-types/